pkg/broker/broker.go

```diff
@@ -355,13 +377,8 @@ func (a AnsibleBroker) Bind(instanceUUID uuid.UUID, bindingUUID uuid.UUID, req *
 	//
 	// See if the service instance still exists, if not send back a badrequest.
 
-	instance, err := a.dao.GetServiceInstance(instanceUUID.String())
+	instance, err := a.GetServiceInstance(instanceUUID)
```

Comment:
Then make it lowercase get. Or remove the receiver and the the a function by passing in the broker.